cdc/sinkv2/txneventsink/txn_event_sink.go

```diff
@@ -23,9 +26,9 @@ import (
 // When we process row events, TableStopped is used to
 // determine if we really need to process the event.
 type TxnEvent struct {
-	Txn          *model.SingleTableTxn
-	Callback     func()
-	TableStopped *atomic.Bool
+	Txn       *model.SingleTableTxn
+	stopped   *atomic.Bool
```

Comment:
Please wrap this in a struct called `tableStatus`. It would be good for extensibility.